src/openrct2/network/Twitch.cpp

```diff
@@ -286,12 +283,12 @@ namespace Twitch
             context->WriteLine("API URL is empty! skipping request...");
             return;
         }
-        snprintf(url, sizeof(url), "%s/channel/%s/audience", gConfigTwitch.api_url, gConfigTwitch.channel);
+        snprintf(url, sizeof(url), requestFormat.c_str(), gConfigTwitch.api_url, gConfigTwitch.channel);
 
         _twitchState = TWITCH_STATE_WAITING;
         _twitchIdle = false;
 
-        Http::DoAsync({ url }, [](Http::Response res) {
+        Http::DoAsync({ url }, [&](Http::Response res) {
```

Comment:
This is for actually modifying `_twitchState`, `_twitchJsonResponse` I presume?

Reply:
Kind of, is for the lambda to have access to `operation` to use it on the assignment

Reply:
It would have had access to `operation` with value capture anyway.

Reply:
Indeed, didn't give too much thought, will change it